Development

diarisdiakite opened the pull request:
In this PR I added:
- Added the Header Section with the subsection nav;
- updated the styles.css file with the nav, nav-link and headline's classes and id #headline-bg

diarisdiakite commented:
In Milestone I:
- Added the font awesome for the header and for social medias
- I fixed the linters CSS issues
- I added the media queries for min-width 755px and 768px

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+
+  <header>
+
+    <div id="settings">
+      <div id="settings-date">
+
+      </div>
+      
+      <div class="settings-fa">
+        <i class="fas fa-signal"></i>
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-battery-full"></i>
+      </div>
+    </div>
```
- [ ] Nice work sticking to the Figma template (literally 😄 ), but this part is a phone screen bar and is not part of the design, please make sure to take it off.  
![2023-03-23_231127](https://user-images.githubusercontent.com/104319462/227385127-9c62ac11-8219-41e0-98d4-c349f7cb1048.png)

diarisdiakite replied:
😄😄😄 Done, thanks @ouasamine, And saying that it took me a lot of time to search them... 😄😄😄

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+      </div>
+    </div>
+
+    <div id="union">
+      <img src="images/Icons/Icon - Menu.png" alt="union">
+    </div>
```
- [ ] This hamburger icon/button should be wrapped inside a `<nav>` tag for semantic reasons, and it will increase the accessibility of your app.

diarisdiakite replied:
Done @ouasamine

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+  </header>
+
+  <section class="headline">
+
+    <div>
+      <img id="headline-bg" src="images/Header-llustration-desktop@2x.png" alt="background-image">
+    </div>
```
- [ ] The `<img>` tag here is wrongly used, this is a background image and is not part of the page content, therefore what should be used to set the background is the CSS `background-image` property instead.

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+        Hey there. 
+        <br>
+        I’m Didy.
+        <br>
+        <span class="green"> I’m a Full Stack developer from Microverse</span>
```
- [ ] This subtitle should have its own `<hx>` you can use `<h2>` for example, and it should be outside the `<h1>` tag.

diarisdiakite replied:
Done!

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+        <br>
+        <span class="green"> I’m a Full Stack developer from Microverse</span>
+      </h1>
+
+      <h2>
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+      </h2>
```
- [ ] This description should be wrapped inside a `<p>` stage, its more semantically proper and will improve your website accessibility.

diarisdiakite replied:
Done!

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+
+    
+  </header>
+
+  <section class="headline">
+
```
- [ ] Nice work on the header section in general, however, there are still some points that need to be worked to match the Figma design 🔽 

## Your Design 
![2023-03-23_232317](https://user-images.githubusercontent.com/104319462/227387650-7cdb95f9-49a9-4f8d-83e7-741009f16145.png)

## Figma Design
![2023-03-23_232733](https://user-images.githubusercontent.com/104319462/227387662-f229a2d2-ffc0-445d-b70e-63ba5056bc32.png)

ouasamine commented on index.html:
```diff
@@ -0,0 +1,77 @@
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+      </h2>
+
+      <div class="social">
+        <a href="https://twitter.com/diarisdiakite">
+          <i class="fab fa-twitter"></i>
+        </a>
+        <a href="https://www.linkedin.com/diarisdiakite/">
+          <i class="fab fa-linkedin-in"></i>
+        </a>
+        <a href="https://www.medium.com/#">
+          <i class="fab fa-medium"></i>
+        </a>
+        <a href="https://www.github.com/diarisdiakite">
+          <i class="fab fa-github"></i>
+        </a>
+        <a href="https://www.angellist.com/#">
+          <i class="fab fa-angellist"></i>
+        </a>
+      </div>
```
- [ ] This is a list of social media links, there what should be used here is a `<ul>` stage instead of `<div>` and the `<a>` tags should be wrapped inside `<li>` tags, it's more semantically proper and will improve accessibility.

diarisdiakite replied:
Done @ouasamine

ouasamine reviewed (changes requested):
Hi @diarisdiakite 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)

## Highlights
- Nice work following the Figma design generally ✔️ 
- Good PR summary 👏🏽 👏🏽 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diarisdiakite commented:
In this Milestone I:
- Deleted the #setting, 
- Added a paragraph tag for the description replaced the span by a <h2> tag,
- Have Set the image as a background-image,
- wrapped the social media list inside a <ul> tag for and wrapped the links inside <li> tags,
- Changed the green color to match the template color,
- wrapped the hamburger button inside a <nav> tag

LionRouge1 reviewed (approved):
Hi @diarisdiakite  , 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
- You've done well implementing the requested changes:+1:
- Your design looks almost identic with Figma's design :1st_place_medal: 

![done](https://media.giphy.com/media/l1uga8ObPBnIBoXUQ/giphy.gif)

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] You are using a README file that is not related to this project. Kindly use the [README template](https://github.com/microverseinc/readme-template) provided by Microverse and remember to customize it to suit your project.
This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you make the change.

- [OPTIONAL] Please in your upcoming project, make sure your Branch has a feature or action name (e.g. 'setup_mobile_first' or 'add_project_files'). This will help you know later what has been done in the branch by also naming your branch after the feature or action implemented there, making you follow the GitHub flow and Gitflow.
This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you make the change.

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

diarisdiakite commented:
In this Milestone I updated the README.md file
